typedb/connection/cluster/server_client.py

```diff
@@ -56,7 +56,7 @@ def _new_channel(self) -> grpc.Channel:
             channel_credentials = grpc.ssl_channel_credentials()
         combined_credentials = grpc.composite_channel_credentials(
             channel_credentials,
-            grpc.metadata_call_credentials(_CredentialAuth(self._credential.username(), self._credential.password()))
+            grpc.metadata_call_credentials(_CredentialAuth(lambda: self._credential, lambda: self._stub.token()))
```

Comment:
The authentication metadata will either contain the credential (eg., username / password), or token.

Reply:
In practice, the credential is only sent during the `user_token_renew` call. Once the `token` has been renewed, it will be preferred.